### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by TramanhLe**
Update functionality currently working with the first name and last name of patient, will continue to work on it.
ISSUE: other fields like address and phone number are unable to update.

### CODE REVIEW

File: client/src/pages/LogIn/LogIn.jsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ const LogIn = (props) => {
               justify="center"
             >
               <Grid item className={classes.logoContainer}>
-                <img src={logo} height="75rem" width="75rem" />
+                <img src={logo} height="75rem" width="95rem" />
```
Comment (coletrumbo-ss) [inline client/src/pages/LogIn/LogIn.jsx:64]: This is a change to the login page. I see how it is related to the Profile page, but this page has seen a bit of action from various people lately. Is Hoff and Daniel both aware of these changes? Did this change come from a merge with master? Or part of your direct edits?

File: client/src/services/LogIn/actions.js [hunk 2]
```diff
@@ -44,9 +44,10 @@ const _loginSuccess = (userInfo) => {
         address: userInfo.address1_line1,
         city: userInfo.address1_city,
         state: userInfo.address1_stateorprovince,
+        zipCode: userInfo.address1_postalcode,
```
Comment (coletrumbo-ss) [inline client/src/services/LogIn/actions.js:47]: needs to be seen by others working on Login or using the login actions.

File: client/src/utils/loginApi.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 import { adalApiFetch } from "../adalConfig";
 import { getConfig } from "./APIHeaders";
-import {buildApiCall} from './helperFunctions'
+import {buildApiCall} from './helperFunctions';
```
Comment (coletrumbo-ss) [inline client/src/utils/loginApi.js:4]: needs to be seen by others working with loginApi

**Review by coletrumbo-ss — CHANGES_REQUESTED** [review]
Since we are not using the /pages directory anymore, could you consolidate everything into the /components?

File: client/src/pages/LogIn/LogIn.jsx (hunk 1 above)
Comment (TramanhLe) [reply client/src/pages/LogIn/LogIn.jsx:64]: I had permission from Haffed to modify the Login page and set it up for dark theming, so this was just a visual edit.

File: client/src/services/LogIn/actions.js (hunk 2 above)
Comment (TramanhLe) [reply client/src/services/LogIn/actions.js:47]: This was a component that no one else is using besides me, or else they would realize that while the API was called correctly, it was mapping incorrectly to userInfo.

**Comment by TramanhLe** [comment]
I will transfer my components from /pages into /component. Thank you for being thorough with this pull request.

**Review by coletrumbo-ss — APPROVED** [review]
Looks good, everything working on my computer

**Pull request merged**